mountpoint-s3-crt/src/common/uri.rs

```diff
@@ -6,7 +6,7 @@ use std::os::unix::prelude::OsStrExt;
 
 use mountpoint_s3_crt_sys::{
     aws_byte_cursor_from_buf, aws_uri, aws_uri_authority, aws_uri_clean_up, aws_uri_host_name, aws_uri_init_parse,
-    aws_uri_path, aws_uri_query_string, aws_uri_scheme,
+    aws_uri_path, aws_uri_query_string, aws_uri_scheme, aws_uri_port
```

Comment:
One more formatter thing:
```suggestion
    aws_uri_path, aws_uri_port, aws_uri_query_string, aws_uri_scheme,
```